docs/conf.py

```diff
@@ -73,6 +74,7 @@
             ("Add new actions", "new-actions"),
             ("Stereo agent", "stereo-agent"),
             ("Notebooks", "notebooks"),
+            ("Image extraction", "image-extractor"),
```

Comment:
I was sure that was already fixed in previous PRs. Make sure you're rebased on latest master.